packages/cardano-graphql/src/Schema/types/Transaction/TransactionOutput.ts

```diff
@@ -0,0 +1,18 @@
+import { Cardano } from '@cardano-sdk/core';
+import { Field, Int, ObjectType } from 'type-graphql';
+import { Transaction } from './Transaction';
+import { Value } from './Value';
+
+@ObjectType()
+export class TransactionOutput {
+  @Field(() => String)
+  address: Cardano.Address;
+  @Field()
+  addressHasScript: boolean; // Review: what's this?
```

Comment:
It's a convenience field from the `cardano-db-sync` model exposed in the GraphQL schema. https://github.com/input-output-hk/cardano-db-sync/blob/master/doc/schema.md#tx_out

Let's omit for now as our model here should result in a natural query to determine this state.